packages/f/font-ubuntu-ttf/files/ubuntu.metainfo.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Solus Developers <copyright@getsol.us> -->
+<component type="font">
+  <id>Ubuntu</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Ubuntu</name>
+  <homepage>https://design.ubuntu.com/font</homepage>
```

Comment:
In this file the format of the homepage needs to be in this format:

```
<url type="homepage">https://design.ubuntu.com/font</url>
```

Reply:
done